python/lsst/ip/diffim/diffimLib.i

```diff
@@ -269,23 +273,8 @@ lsst::afw::image::Image<PIXTYPE>
 
 %include "lsst/ip/diffim/BasisLists.h"
 
-/******************************************************************************/
-
-%{
-#include "lsst/ip/diffim/DipoleAlgorithms.h"
-%}
 
-%shared_ptr(lsst::ip::diffim::DipoleCentroidControl)
-%shared_ptr(lsst::ip::diffim::DipoleFluxControl)
-%shared_ptr(lsst::ip::diffim::DipoleCentroidAlgorithm)
-%shared_ptr(lsst::ip::diffim::DipoleCentroidControl)
-%shared_ptr(lsst::ip::diffim::DipoleFluxAlgorithm)
-%shared_ptr(lsst::ip::diffim::DipoleFluxControl)
-%shared_ptr(lsst::ip::diffim::NaiveDipoleCentroidControl)
-%shared_ptr(lsst::ip::diffim::NaiveDipoleFluxControl)
-%shared_ptr(lsst::ip::diffim::PsfDipoleFluxControl)
-
-%include "lsst/ip/diffim/DipoleAlgorithms.h"
+%include  "lsst/ip/diffim/dipole.i"
```

Comment:
There seems to be significant duplicated code between dipole.i and this file.  Is it worth putting these few lines in a new file if it also requires a bunch of boilerplate as well?